data/Ewaaugustine.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "Ewa Augustine",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Student Doctor turned **Software Developer**. Lover of good things and a technical writer",
+  "avatar": "https://github.com/Ewaaugustine.png"
```

Comment:
```suggestion
  "avatar": "https://github.com/Ewaaugustine.png",
```

Reply:
We no longer need the `avatar` key as of our latest update.